### PR TITLE
Cumulocity Documentation Review

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-applications.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ helpcontent:
...
-    
+
     **Custom applications** - Applications owned by the tenant. You can add custom applications in various ways as own applications.
 
 
     Click on an application to view the application properties. To add an application, click **Add application** and follow the instructions in the wizard, see also the *User guide*."
 ---
 
-The {{< product-c8y-iot >}} platform distinguishes between applications and microservices, see also [Developing applications](concepts/applications/#overview) in the *Concepts guide*..
+The {{< product-c8y-iot >}} platform distinguishes between applications and microservices, see also [Developing applications](/concepts/applications) in the *Concepts guide*..
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:19]: remove one full stop at the end

File: content/users-guide/device-management-bundle/monitoring-device-services.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ See [Working with alarms](#alarm-monitoring) for detailed information on alarms.
 
 >**Info:** The service details **Alarms** tab displays only alarms which have the particular service as a source. It does not display any alarms sourced by the device itself.
 
-<a name="events"></a>
+<a name="service-events"></a>
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/monitoring-device-services.md:48]: @JoshuaAdemola Did you check throughout the documentation where this anchor is referenced?

File: content/welcome/additional-resources.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ aliases:
 
 ### Installation and operations documentation
 
-The **Installation and Operations guides** for the {{< product-c8y-iot >}} platform are not publicly available. They are provided on the [{{< company-sag >}} {{< sag-portal >}}]({{< link-sag-documentation >}}). Search for "{{< product-c8y-iot >}} Platform" as product. To view the Installation and Operations guides you must click the PDF icon for the version of your choice.
+The **Installation and Operations guides** for the {{< product-c8y-iot >}} platform are not publicly available. They are provided on the [{{< company-sag >}} {{< sag-documentation >}}]({{< link-sag-documentation >}}) website. Search for "{{< product-c8y-iot >}} Platform" as product. To view the Installation and Operations guides you must click the PDF icon for the version of your choice.
```
Comment (BeateRixen) [inline content/welcome/additional-resources.md:11]: @Why did you change this shortcode?

File: content/users-guide/device-management-bundle/monitoring-device-services.md (hunk 2 above)
Comment (JoshuaAdemola) [reply content/users-guide/device-management-bundle/monitoring-device-services.md:48]: Yes I did. There is another anchor called events so it creates a conflict. There is only one link that refers to this anchor `<a name="service-events"></a>`.

File: content/welcome/additional-resources.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/welcome/additional-resources.md]: Fixed

File: content/microservice-sdk/rest-bundle/microservice-development.md [hunk 4]
```diff
@@ -526,7 +526,7 @@ A status of PENDING means here that the device has not yet picked up the operati
 
 #### Listening for events
 
-Besides querying the {{< product-c8y-iot >}} data store, you can also process and receive events in real time as described in [Real-time processing in {{< product-c8y-iot >}}](/concepts/realtime). For example, assume that you would like to display real-time location updates on a map. Use the Administration application (or the [REST API](/reference/real-time-statements)) to create a new rule module "myRule":
+Besides querying the {{< product-c8y-iot >}} data store, you can also process and receive events in real time as described in [Real-time processing in {{< product-c8y-iot >}}](/concepts/realtime). For example, assume that you would like to display real-time location updates on a map. Use the Administration application (or the [REST API](https://cumulocity.com/guides/10.9.0/event-language/real-time-statements/)) to create a new rule module "myRule":
```
Comment (BeateRixen) [inline content/microservice-sdk/rest-bundle/microservice-development.md:529]: cumulocity.com should probably be a shortcode?

Comment (JoshuaAdemola) [reply content/microservice-sdk/rest-bundle/microservice-development.md]: Fixes

File: content/device-tutorials/netcomm-router-bundle/release-history.md [hunk 5]
```diff
@@ -7,13 +7,10 @@ weight: 60
...
 
-{{< c8y-admon-info >}}
-If you are using either [NetComm Wireless NTC-6200](https://support.netcommwireless.com/products/NTC-6200%20Series) or [NetComm Wireless NTC-140W](https://support.netcommwireless.com/products/NTC-140W%20Series), you must use our prior NetComm Agent version 4.2.3 or older. Contact [product support](/welcome/contacting-support/) or refer to the [NetComm](https://{{< domain-c8y >}}/guides/10.4.6/devices/netcommwireless/) section in the Devices guide in the {{< product-c8y-iot >}} 10.4.6 documentation 10.4.6 for information on how to get the packages.
-{{< /c8y-admon-info >}}
+> **Info:** If you are using either [NetComm Wireless NTC-6200](https://support.netcommwireless.com/products/NTC-6200-01) or [NetComm Wireless NTC-140W](https://support.netcommwireless.com/products/NTC-140W%20Series), you must use our prior NetComm Agent version 4.2.3 or older. Contact [product support](/welcome/contacting-support/) or refer to the [NetComm](https://{{< domain-c8y >}}/guides/10.4.6/devices/netcommwireless/) section in the Devices guide in the {{< product-c8y-iot >}} 10.4.6 documentation 10.4.6 for information on how to get the packages.
```
Comment (BeateRixen) [inline content/device-tutorials/netcomm-router-bundle/release-history.md:10]: It seems that you have now overwritten your admonition changes?

**Review by MWindel — APPROVED** [review]
